chart/compass/charts/kyma-environment-broker/templates/virtualservice.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: {{ include "kyma-env-broker.fullname" . }}
+  labels:
+{{ include "kyma-env-broker.labels" . | indent 4 }}
+spec:
+  gateways:
+  - compass-istio-gateway.compass-system.svc.cluster.local
```

Comment:
shouldn't it be: `{{ .Values.global.gateway.tls.host }}.{{ .Values.global.ingress.domainName }}`?

I took that from `compass/charts/gateway/templates/virtualservice.yaml`